### DISCUSSION TITLE
aiohttp.server

### DISCUSSION THREAD

**Discussion by akeslo** [home-assistant/operating-system#2241] [Q&A]
How do I monitor any type of traffic request to the HA aiohttp.server? I would like the ability to see any attempts and not just failures which do currently appear in the HA supervisor log.

Thanks!

**Comment by agners** [comment]
There is no traffic monitoring built-in. This is more about discussions of Home Assistant OS itself.

It can maybe be done with a custom add-on (with Wireshark or similar preinstalled) which uses host network access.
